1_fetch/in/Legend_FORESCE_Land_Cover.csv

```diff
@@ -0,0 +1,22 @@
+FORESCE_value,FORESCE_description,Reclassify_match,Reclassify_description
+0,Corn (no data),3,Cultivated Crops
+1,Hay / alfalfa,3,Cultivated Crops
+2,Corn,3,Cultivated Crops
+3,Soybeans,3,Cultivated Crops
+4,Fallow/Idle,2,Pasture/Hay
+5,Wheat,3,Cultivated Crops
+6,Fruits / Vegetables,3,Cultivated Crops
+7,Tobacco,3,Cultivated Crops
+8,Other Crops,3,Cultivated Crops
+9,Open Water,1,Water
+10,Developed Low Intensity,4,Developed Areas
+11,Developed Med-High Intensity,4,Developed Areas
```

Comment:
We discussed keeping these as separate classes [here](https://github.com/USGS-R/drb-inland-salinity-ml/issues/55#issuecomment-1017756761). Just making sure that's on your to-do for this dataset. You can use 41 and 42 as sub-class numbers if you want to keep it numeric. That also is in line with how NLCD assigns numbers to sub-classes

Reply:
right, I'll fix that. =)